tests/functional/ftp/test_api.py

```diff
@@ -43,7 +43,7 @@ def test_list_folders():
 
 
 def test_delete():
-    source = f"ftp://hostname/source.txt"
+    source = "ftp://hostname/source.txt"
```

Comment:
did you catch this eyeballing or you have a tool check f-strings? It gets to my nerves that flake8 doesn't properly analyze f-strings

Reply:
flake8 >=3.8 catches by default with [F541](https://flake8.pycqa.org/en/latest/user/error-codes.html)